### ISSUE TITLE
Branch to 6.5 

### ISSUE THREAD

**Issue by san7ket** [SatelliteQE/airgun#317]
We will be soon starting with 6.6. When would be the best time to branch ?
@abalakh @ldjebran @mirzal @oshtaier

**Comment by mirzal** [comment]
My opinion when looking at open PRs and issues:

* #276 - would be nice to merge before branch
* #284 - seems no test actually require it and there is some disagreement; conversation has stalled and it would be nice to revive it, but can be done after branch
* #269 and #309 - look like responsible for some test failures, but afaik nobody is actively working on these
* #198 and #223 - pretty much declined and afaik nobody is working of them; I'm not even sure why they are still open


So, I would wait at least couple of days to see if we can get #276 in before branch. #269 and #309 would be nice, too, but since nobody seems to be working on them, I wouldn't let them block branch.

**Comment by mirzal** [comment]
#276 is now merged, #284 is still stalled.

Now that Robottelo 6.5.z is branched, it's probably good time to branch Airgun as well.
@san7ket how should we approach this? Is there any coordination needed? Any changes in Jenkins jobs or automation tools that should be merged first? Or should be merged soon after Airgun branch?

**Comment by san7ket** [comment]
Branch here and update the robottelo's 6.5.z branch https://github.com/SatelliteQE/robottelo/blob/6.5.z/requirements.txt#L33

Don't need any changes to jobs

**Comment by mirzal** [comment]
@san7ket thanks for confirmation.

Airgun branch created. I also created new PR for Robottelo: https://github.com/SatelliteQE/robottelo/pull/6896

**Comment by mirzal** [comment]
Robottelo PR merged, so I'm closing this.
